config/hardware/addons/accelerometers/adxl345_ebb.cfg

```diff
@@ -0,0 +1,17 @@
+[adxl345]
+cs_pin: EBBCan: ADXL_CS
+spi_software_sclk_pin: EBBCan: ADXL_SCLK
+spi_software_mosi_pin: EBBCan: ADXL_MOSI
+spi_software_miso_pin: EBBCan: ADXL_MISO
```

Comment:
Can you change EBBCan by "toolhead". I think, this will allow more genericity.

Reply:
You're right, changes done.